app/com/linkedin/drelephant/tuning/FitnessComputeUtil.java

```diff
@@ -430,6 +431,16 @@ private void updateExecutionMetrics(List<TuningJobExecutionParamSet> completedJo
     logger.info("Execution metrics updated");
   }
 
+  private void parameterSpaceOptimization(JobSuggestedParamSet jobSuggestedParamSet, JobExecution jobExecution,
```

Comment:
This method should be part of AutoTuningOptimizeManager.

Reply:
why ? this is general method , calling optimize-manager based on algorithm.  Then extractParameterInformation and parameterOptimizer are part of AutoTuningOptimizeManager . Same is done in other places